[12/Oct/2014 15:22:02] WARNING [django.request:143] Not Found: /favicon.ico
[12/Oct/2014 15:23:06] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 98, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 340, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 224, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 231, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 17
    processoren = Processoren.objects[]
                                      ^
SyntaxError: invalid syntax
[12/Oct/2014 15:25:59] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 98, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 340, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 224, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 231, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 17
    processoren = Processoren.objects[:10, categorie="Processor"]
                                                    ^
SyntaxError: invalid syntax
[12/Oct/2014 15:27:09] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 98, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 340, in resolve
    sub_match = pattern.resolve(new_path)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 224, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 231, in callback
    self._callback = get_callable(self._callback_str)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 97, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 17
    processoren = Processoren.objects[:10 | categorie="Processor"]
                                                     ^
SyntaxError: invalid syntax
[12/Oct/2014 15:34:44] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 17, in processoren
    processoren = Processoren.objects(categorie.contains('Processor'))[:10]
NameError: global name 'categorie' is not defined
[13/Oct/2014 10:27:00] WARNING [django.request:143] Not Found: /favicon.ico
[13/Oct/2014 10:27:00] WARNING [django.request:143] Not Found: /favicon.ico
[13/Oct/2014 11:35:59] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: ' processor.naam' from 'u processor.naam'
[13/Oct/2014 11:47:05] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 17, in processoren
    processoren = Processoren.objects(categorie__contains='Processor')[:10]
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 125, in __getitem__
    queryset._cursor_obj = queryset._cursor[key]
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 1182, in _cursor
    self._cursor_obj = self._collection.find(self._query,
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 1215, in _query
    self._mongo_query = self._query_obj.to_query(self._document)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 92, in to_query
    query = query.accept(QueryCompilerVisitor(document))
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 157, in accept
    return visitor.visit_query(self)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 80, in visit_query
    return transform.query(self.document, **query.query)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/transform.py", line 59, in query
    raise InvalidQueryError(e)
InvalidQueryError: Cannot resolve field "categorie"
[13/Oct/2014 11:47:43] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 17, in processoren
    processoren = Processoren.objects(categorie__contains='Processor')[:10]
TypeError: 'Manager' object is not callable
[13/Oct/2014 12:18:55] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 573, in __init__
    var_obj = Variable(var)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 727, in __init__
    var)
TemplateSyntaxError: Variables and attributes may not begin with underscores: '__unicode__'
[13/Oct/2014 12:23:30] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    return  self.response.write('processoren.html', {'Processoren': processoren},
NameError: global name 'self' is not defined
[13/Oct/2014 12:23:44] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    return  request.response.write('processoren.html', {'Processoren': processoren},
AttributeError: 'WSGIRequest' object has no attribute 'response'
[13/Oct/2014 12:24:35] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    return  request.response.write('processoren.html', {'Processoren': processoren},
AttributeError: 'WSGIRequest' object has no attribute 'response'
[13/Oct/2014 12:27:18] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    return respone.write('processoren.html', {'Processoren': processoren},
NameError: global name 'respone' is not defined
[13/Oct/2014 12:27:26] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    return response.write('processoren.html', {'Processoren': processoren},
NameError: global name 'response' is not defined
[13/Oct/2014 12:27:33] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 19, in processoren
    return self.response.write('processoren.html', {'Processoren': processoren},
NameError: global name 'self' is not defined
[13/Oct/2014 12:31:13] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '[0]' from 'processor.naam[0]'
[13/Oct/2014 14:12:18] WARNING [django.request:143] Not Found: /favicon.ico
[13/Oct/2014 14:12:18] WARNING [django.request:143] Not Found: /favicon.ico
[13/Oct/2014 14:12:19] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 18, in processoren
    processoren = Processoren.objects(categorie__contains='Processor', ensure_ascii=False)[:10]
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 125, in __getitem__
    queryset._cursor_obj = queryset._cursor[key]
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 1182, in _cursor
    self._cursor_obj = self._collection.find(self._query,
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 1215, in _query
    self._mongo_query = self._query_obj.to_query(self._document)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 92, in to_query
    query = query.accept(QueryCompilerVisitor(document))
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 157, in accept
    return visitor.visit_query(self)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 80, in visit_query
    return transform.query(self.document, **query.query)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/transform.py", line 59, in query
    raise InvalidQueryError(e)
InvalidQueryError: Cannot resolve field "ensure_ascii"
[13/Oct/2014 14:17:33] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 18, in processoren
    processoren = Processoren.objects(categorie__contains='Processor', ensure_ascii=False)[:10]
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 125, in __getitem__
    queryset._cursor_obj = queryset._cursor[key]
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 1182, in _cursor
    self._cursor_obj = self._collection.find(self._query,
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/base.py", line 1215, in _query
    self._mongo_query = self._query_obj.to_query(self._document)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 92, in to_query
    query = query.accept(QueryCompilerVisitor(document))
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 157, in accept
    return visitor.visit_query(self)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/visitor.py", line 80, in visit_query
    return transform.query(self.document, **query.query)
  File "/usr/local/lib/python2.7/dist-packages/mongoengine/queryset/transform.py", line 59, in query
    raise InvalidQueryError(e)
InvalidQueryError: Cannot resolve field "ensure_ascii"
[13/Oct/2014 14:24:01] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '[0].encode('utf8')' from 'processor.naa[0].encode('utf8')'
[13/Oct/2014 14:24:14] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '[0].encode('utf8')' from 'processor.naam[0].encode('utf8')'
[13/Oct/2014 14:24:42] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '[0]utf8')' from 'processor.naam[0]utf8')'
[13/Oct/2014 14:25:18] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '[0]' from 'processor.naam[0]'
[13/Oct/2014 14:25:31] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '('utf8')' from 'processo.encode('utf8')'
[13/Oct/2014 14:25:55] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '[0]' from 'processor.naam[0]'
[13/Oct/2014 14:26:15] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '("utf8")' from 'processor.naam.encode("utf8")'
[13/Oct/2014 14:27:05] ERROR [django.request:231] Internal Server Error: /processoren/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/robin/Work/School/A-Pc/Website/APc/pcbuilder/views.py", line 20, in processoren
    context_instance=RequestContext(request))
  File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 170, in render_to_string
    t = get_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 833, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 266, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 372, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 588, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '(processor.naam)' from 'str(processor.naam)'
